chromium_src/chrome/browser/ui/views/bubble/bubble_contents_wrapper.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <utility>
+
+#include "chrome/browser/ui/views/bubble/bubble_contents_wrapper.h"
+#include "components/sessions/content/session_tab_helper.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/browser/web_contents_delegate.h"
+#include "content/public/browser/web_contents_observer.h"
+
+// Trezor library opens a popup window to connect with a device.
```

Comment:
this comment should go https://github.com/brave/brave-core/pull/10902/files#diff-496bd71e0557ff2b87c0654e9ab1ae949f48a2dc61bef1235cb85e5bcac730edR69 because it's specific to the wallet code.

Reply:
done

Reply:
looks like it's still here, did you forget to push? Some generic comments are good here, but trezor specific comments would go in wallet code